tools/windows-node-installer/README.md

```diff
@@ -62,3 +67,26 @@ The IDs of created instance and security group are saved to the `windows-node-in
 The `wni` destroys all resources (instances and security groups) specified in the `windows-node-installer.json` file. 
 Security groups will not be deleted if they are still in-use by other instances.
 
+
+### Azure Platform
```

Comment:
Why is there an azure subsection but no aws section? Is this something that will be changed in a future commit? To me it seems like there is a lot of duplicate information here.